backend/app/services/db/profile_manager.py

```diff
@@ -0,0 +1,189 @@
+from .db_connect import connect
+from .user_manager import get_user
+
+
+def create_profile(username: str, profile_description: str = None, profile_picture: str = None,
+                   interest1: str = None, interest2: str = None, interest3: str = None,
+                   gender: str = None):
+    """Creates a profile with the given value.
+
+    Assumes that a user is created first. A user should be inserted into

```

Comment:
how does this make sense, you should sign up to become a user in the same place you create the profile. the user cant be created first because the profile should just generate right after signing up. it should all be in one sign up page instead of signing up as a user then creating your profile.